### DISCUSSION TITLE
Email deduping (.eml files) - optimal settings?  (text file matching)

### DISCUSSION THREAD

**Discussion by R-T-R-T** [hsoft/dupeguru#1058] [Q&A] [answered]
I've got a single folder (no subfolders) with duplicate emails in it, all files are in .eml format.

File names are of course different, but the criteria to match on need to be exactly the same in the duplicate files: file size and the file's text contents.

What are the optimal settings I should set in dupeGuru? I see lots of tweaks for photo and music matching, I want a 100% exact match on a text file where only the file name is different.

I'm guessing:
Application Mode: Standard, Scan Type: Contents, More Options: _THIS IS WHERE I'M STUCK_

I probably want to uncheck "Ignore files smaller than" and "Can mix file kind," but what about the Filter Hardness setting, and "use regular expressions when filtering?" Anything else to change from the defaults?

Also, I've seen some people comment on Czkawka (https://github.com/qarmin/czkawka). For text file matching, is this a better option?

_Thank you in advance for any direction you can give!_

**Comment by arsenetar** [comment, marked as answer]
If for identical emails you expect the contents to be the same you should just be able to run a contents scan.  If your files are small you will want to uncheck the "Ignore files smaller than" or adjust the value as you have stated. The filter does not apply to a contents scan.

**Reply by R-T-R-T** [reply]
The filter hardness number setting you've set in the other modes carries over to a Contents scan, but you're right, it's not changeable in a Contents scan. I was wondering about that. So it's just a user interface thing, should be removed. That was throwing me off.

Thank you!
